src/main/java/carpetextra/utils/ChunkUtils.java

```diff
@@ -12,8 +12,9 @@
 public class ChunkUtils
 {
     public static final ChunkTicketType<ChunkPos> ENDER_PEARL_TICKET = ChunkTicketType.create("ender_pearl", Comparator.comparingLong(ChunkPos::toLong), 2);
+    public static final ChunkTicketType<ChunkPos> MINECART_TICKET = ChunkTicketType.create("minecart", Comparator.comparingLong(ChunkPos::toLong), 2);
 
-    public static void addEnderPearlChunkTicket(Entity entity)
+    public static void addCustomChunkTicket(Entity entity, ChunkTicketType<ChunkPos> ticketType)
```

Comment:
Should likely be renamed to indicate it'll only add the ticket if the entity is moving.